src/db/migrations/postgres/CreateCoinAndCoinPriceTables1631234567894.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateCoinAndCoinPriceTables1631234567894 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE coin (
+        id serial PRIMARY KEY,
+        name varchar
+      )
+    `);
+
+    await queryRunner.query(`
+      CREATE TABLE coin_price (
+        id serial PRIMARY KEY,
+        coinId integer,
+        price numeric,
+        FOREIGN KEY (coinId) REFERENCES coin (id) ON DELETE CASCADE
```

Comment:
Again, this is the right syntax but has overhead.
Feel free to as me why this is the best practice in our case!

Reply:
sure, thanks. done.